packages/neos-ui/package.json

```diff
@@ -58,6 +58,7 @@
     "react-dnd": "^10.0.0",
     "react-dnd-html5-backend": "^10.0.0",
     "react-dom": "^16.12.0",
+    "react-error-boundary": "^4.0.4",
```

Comment:
Why don't you use the `ErrorBoundary` of React itself?

Reply:
i was unsure at first too but then i remembered that @grebaldi did use this lib also in Sitegeist.Archaeopteryx https://github.com/sitegeist/Sitegeist.Archaeopteryx/blob/a003751455ccd6759cd6968dea11c08481ee4296/Neos.Ui/error-handling/src/application/ErrorBoundary.tsx and i thought it is good practice.

This lib will allow dismissing of errors to and stuff like this in the future

Reply:
I still don't see what's the benefit to the default ErrorBoundary?

Reply:
IIRC React didn't have a default Error Boundary when I built Archaeopteryx. Error boundaries were a new concept back then.

I'd definitely opt for official API rather than dependency - so if React provides something, then I'd recommend removing the dependency to `react-error-boundary`.

Reply:
> There is no direct equivalent for static getDerivedStateFromError in function components yet. If you’d like to avoid creating class components, write a single ErrorBoundary component like above and use it throughout your app. Alternatively, use the [react-error-boundary](https://github.com/bvaughn/react-error-boundary) package which does that.

hu interesting ^^

Reply:
Yes, that's the one I built for the Media.UI https://github.com/Flowpack/media-ui/blob/main/Resources/Private/JavaScript/media-module/src/components/ErrorBoundary.tsx

Reply:
adjusted ;) THanks